File: eNote/eNote.Model/Enums/VrstaInstrumenta.cs
using System.Text.Json.Serialization;

namespace eNote.Model.Enums
{
    [JsonConverter(typeof(JsonStringEnumConverter))]
    public enum VrstaInstrumenta
    {
        Zicani = 1,
        Limeni = 2,
        Udaraljke = 3,
        Tipke = 4,
        Elektronicki = 5
    }
}

File: eNote/UI/enote_desktop/lib/models/enums/vrsta_instrumenta.dart
// ignore_for_file: constant_identifier_names

import 'package:json_annotation/json_annotation.dart';

part 'vrsta_instrumenta.g.dart';

@JsonEnum(alwaysCreate: true)
enum VrstaInstrumenta { Zicani, Limeni, Udaraljke, Tipke, Elektronicki }
